src/output-json-ike.c

```diff
@@ -135,12 +152,14 @@ static TmEcode JsonIKELogThreadInit(ThreadVars *t, const void *initdata, void **
 
     if (initdata == NULL) {
         SCLogDebug("Error getting context for EveLogIKE.  \"initdata\" is NULL.");
-        goto error_exit;
+        SCFree(thread);
```

Comment:
Why this change ?
Was this a bug in IKEv2 ?

Reply:
hmmm not sure, from having a look in the code both should work, can throw a few of my pcaps against it to test this change.

Reply:
I would rather keep the code unmodified if there is no problem